en/sensor/inertial_navigation_systems.md

```diff
@@ -0,0 +1,31 @@
+# IMU, AHRS, and INS

```

Comment:
FYI, this is a parent page now. In case we end up with other systems like VectorNav that can act as external INS. I've also used this as a place for a glossary of terms.